Loop.xcodeproj/project.pbxproj

```diff
@@ -2445,6 +2449,23 @@
 			shellPath = /bin/sh;
 			shellScript = "${SRCROOT}/Scripts/copy-frameworks.sh\n";
 		};
+		470A9D15233DAEED008DA68E /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = " #!/usr/bin/env bash\n\nAPP_PATH=\"${TARGET_BUILD_DIR}/${WRAPPER_NAME}\"\n\n# This script loops through the frameworks embedded in the application and\n# removes x86_64 and i386 architectures.\nfind \"$APP_PATH\" -name '*.framework' -type d | while read -r FRAMEWORK\ndo\n    FRAMEWORK_EXECUTABLE_NAME=$(defaults read \"$FRAMEWORK/Info.plist\" CFBundleExecutable)\n    FRAMEWORK_EXECUTABLE_PATH=\"$FRAMEWORK/$FRAMEWORK_EXECUTABLE_NAME\"\n\n    if [ ! -f \"${FRAMEWORK_EXECUTABLE_PATH}\" ]; then\n        continue\n    fi\n\n    echo \"Thinning framework $FRAMEWORK_EXECUTABLE_NAME\"\n    \n    if xcrun lipo -info \"${FRAMEWORK_EXECUTABLE_PATH}\" | grep --silent \"Non-fat\"; then\n        echo \"   Framework non-fat, skipping\"\n        continue\n    fi\n\n    ARCHS=$(lipo -archs \"$FRAMEWORK_EXECUTABLE_PATH\")\n\n    for ARCH in $ARCHS\n    do\n       if [ \"$ARCH\" == \"x86_64\" ] || [ \"$ARCH\" == \"i386\" ]; then\n        echo \"   Removing $ARCH from $FRAMEWORK_EXECUTABLE_NAME\"\n        xcrun lipo -remove \"$ARCH\" \"$FRAMEWORK_EXECUTABLE_PATH\" -o \"$FRAMEWORK_EXECUTABLE_PATH\"\n       fi\n    done\ndone\n";
```

Comment:
Please split this out into a separate script.

Reply:
New script created.